### PR TITLE
updated metric definition and supported roles related information for…

### PR DESCRIPTION
**Pull request by ivlists**
… TAC usage reports

updated metric definition and supported roles related information for TAC usage reports

@JohanFreelancer9

### CODE REVIEW

File: Teams/includes/teams-reports-definitions.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ The Teams reports show data for active users and active channels. For example, i
 
 |Item  |Definition  |
 |---------|---------|
-|Active user     |Measures the number of unique users who perform an action in Teams during the specified date range.    |
+|Active user     |Measures the number of unique users who perform an action in Teams during the specified date range. for user activity report, This represent active users across tenant. However for Teams usage report, This represent active users only within active Teams    |
```
Comment (JohanFreelancer9) [inline Teams/includes/teams-reports-definitions.md:7]:
```suggestion
|Active user     |Measures the number of unique users who perform an action in Teams during the specified date range for the user activity report, This represents active users across the tenant. However, for the Teams usage report, it represents active users only within active Teams.    |
```

File: Teams/teams-analytics-and-reports/teams-reporting-reference.md [hunk 2]
```diff
@@ -26,7 +26,14 @@ A new analytics and reporting experience for Microsoft Teams is available in the
 
 ## How to access the reports
 
-To access the reports, you must be a global admin in Microsoft 365 or Office 365, global reader in Microsoft 365 or Office 365, Teams service admin, or Skype for Business admin. To learn more about Teams admin roles and which reports each admin role can access, see [Use Teams administrator roles to manage Teams](../using-admin-roles.md).
+To access the reports, you must be one of the following role
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-reporting-reference.md:29]:
```suggestion
To access the reports, you must be assigned one of the following roles:
```

File: Teams/teams-analytics-and-reports/teams-reporting-reference.md [hunk 3]
```diff
@@ -26,7 +26,14 @@ A new analytics and reporting experience for Microsoft Teams is available in the
...
+
+- Global administrator
+- Teams/SFB administrator
+- Global reader (Only tenant level aggregate and no per user/team data)
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-reporting-reference.md:33]:
```suggestion
- Teams/Skype for Business administrator
- Global reader (only tenant-level aggregate and no per-user or team data)
```

File: Teams/teams-analytics-and-reports/teams-reporting-reference.md [hunk 4]
```diff
@@ -26,7 +26,14 @@ A new analytics and reporting experience for Microsoft Teams is available in the
...
+
+To learn more about Teams admin roles and which reports each admin role can access, see [Use Teams administrator roles to manage Teams](../using-admin-roles.md).
+
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-reporting-reference.md:36]:
```suggestion
```

File: Teams/teams-analytics-and-reports/teams-usage-report.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ ms.custom: seo-marvel-apr2020
 ---
 # Microsoft Teams usage report
 
-The Teams usage report in the Microsoft Teams admin center gives you an overview of the usage activity in Teams, including the number of active users and channels, so you can quickly see how many users across your organization are using Teams to communicate and collaborate. You can view usage information for  teams, including the number of active users and channels, guests, and messages in each team.
+The Teams usage report in the Microsoft Teams admin center gives you an overview of the usage activity in Teams, including the number of active users and channels, so you can quickly see how many users across your organization are using a perticular team to communicate and collaborate. You can view usage information for  teams, including the number of active users and channels, guests, and messages in each team.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-usage-report.md:23]:
```suggestion
The Teams usage report in the Microsoft Teams admin center gives you an overview of the usage activity in Teams, including the number of active users and channels, so you can quickly see how many users across your organization are using a particular team to communicate and collaborate. You can view usage information for teams, including the number of active users and channels, guests, and messages in each team.
```

File: Teams/teams-analytics-and-reports/teams-usage-report.md [hunk 6]
```diff
@@ -39,9 +39,9 @@ You must be either a global admin, global reader, or Teams service admin to view
...
...s. </li><li>**Active shared channels** (new) is the number of shared channels in a team that has at least one active internal or external user in the specified time period. </li> <li>**Total organized meetings** is the sum of one time scheduled, recurring, unplanned, and unclassified meetings organized in Teams channel during the specified time period </li><li>**Posts** is the number of all the post messages in channels in the specified time period.</li> <li>**Replies** is the number  of all the reply messages in channels in the specified time period.</li> <li>**Mentions** is the number of all the mentions used in messages in the specified time period.</li><li>**Reactions** is the number  of all the reactions to messages in the specified time period.</li><li>**Urgent messages** is the number of all the urgent messages in the specified time period.</li><li>**Channel messages** is the number of unique messages that the team's users posted in team chats during the specified time period.</li> </li> </ul>Note that if a team no longer exists, the name is displayed as "--" in the table. <br><br>To see the information that you want in the table, make sure to add the columns to the table. |
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-usage-report.md:44]:
```suggestion
|**5**   |The table gives you a breakdown of usage by team. <ul><li>**Team name** is the display name of the team. You can click the team name to go to the team's settings page in the Microsoft Teams admin center. </li> <li>**Team ID** is the unique team identifier. </li> <li>**Type** refers to whether the team is a private team or public team  .</li> <li>**Internal active users** is the number of active users including guests who perform an action in that team in the specified time period. <br/>Internal users and guests reside in the same tenant but are not the same.</li><li>**Active guests** is the number of guests who perform an action in that team in the specified time period.</li> <li>**External active users** (new) is the number of active users from external organizations who perform an action in that team in a resource - such as a shared channel in a team. <br/> External users have their own identities in different tenants and are not the same as a guest account.</li><li>**Active channels** is the number of channels in the team that has at least one active user in the specified time period. This includes private, public, and shared channels. </li><li>**Active shared channels** (new) is the number of shared channels in a team that has at least one active internal or external user in the specified time period. </li> <li>**Total organized meetings** is the sum of one time scheduled, recurring, unplanned, and unclassified meetings organized in a Teams channel during the specified time period </li><li>**Posts** is the number of all the post messages in channels in the specified time period.</li> <li>**Replies** is the number of all the reply messages in channels in the specified time period.</li> <li>**Mentions** is the number of all the mentions used in messages in the specified time period.</li><li>**Reactions** is the number of all the reactions to messages in the specified time period.</li><li>**Urgent messages** is the number of all the urgent messages in the specified time period.</li><li>**Channel messages** is the number of unique messages that the team's users posted in team chats during the specified time period.</li> </li> </ul>Note that if a team no longer exists, the name is displayed as "--" in the table. <br><br>To see the information that you want in the table, make sure to add the columns to the table. |
```

File: Teams/teams-analytics-and-reports/teams-usage-report.md [hunk 7]
```diff
@@ -52,7 +52,7 @@ You must be either a global admin, global reader, or Teams service admin to view
 
 ## Make the user specific data anonymous
 
-To make the data in Teams user activity report anonymous, you have to be a global administrator. The global administrator can hide identifiable information (using MD5 hashes) such as display name, group name, email, and AAD ID in the report and its export.
+To make the data in Teams usage report report anonymous, you have to be a global administrator. The global administrator can hide identifiable information (using MD5 hashes) such as display name, group name, email, and AAD ID in the report and its export.
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-usage-report.md:55]:
```suggestion
To make the data in a Teams usage report anonymous, you have to be a Global administrator. The Global administrator can hide identifiable information (using MD5 hashes) such as display name, group name, email, and AAD ID in the report and its export.
```

**Review by ivlists — COMMENTED** [review]
Updated the copy suggestions.

**Comment by ivlists** [comment]
@JohanFreelancer9 I have updated commit to include suggested copy changes.

File: Teams/includes/teams-reports-definitions.md [hunk 8]
```diff
@@ -4,5 +4,5 @@ The Teams reports show data for active users and active channels. For example, i
 
 |Item  |Definition  |
 |---------|---------|
-|Active user     |Measures the number of unique users who perform an action in Teams during the specified date range.    |
+|Active user     |Measures the number of unique users who perform an action in Teams during the specified date range for the user activity report, This represents active users across the tenant. However, for the Teams usage report, it represents active users only within active Teams.    |
```
Comment (JohanFreelancer9) [inline Teams/includes/teams-reports-definitions.md:7]:
```suggestion
|Active user     |Measures the number of unique users who perform an action in Teams during the specified date range for the user activity report. This represents active users across the tenant. However, for the Teams usage report, it represents active users only within active Teams.    |
```

File: Teams/teams-analytics-and-reports/teams-reporting-reference.md [hunk 9]
```diff
@@ -26,7 +26,13 @@ A new analytics and reporting experience for Microsoft Teams is available in the
 
 ## How to access the reports
 
-To access the reports, you must be a global admin in Microsoft 365 or Office 365, global reader in Microsoft 365 or Office 365, Teams service admin, or Skype for Business admin. To learn more about Teams admin roles and which reports each admin role can access, see [Use Teams administrator roles to manage Teams](../using-admin-roles.md).
+To access the reports, you must be assigned one of the following roles:
+
+- Global administrator
+- Teams/Skype for Business administrator
+- Global reader (only tenant-level aggregate and no per-user or team data)
```
Comment (JohanFreelancer9) [inline Teams/teams-analytics-and-reports/teams-reporting-reference.md:33]:
```suggestion
- Global reader (only tenant-level aggregate and no per-user or Teams data)
```

**Comment by ivlists** [comment]
This pull request is closed without notice @scanum . Can we please reopen and merge it. ?

**Comment by scanum** [comment]
Hi @ivlists hope all is well. This pull request was closed since we did not hear back from you since November 30, 2022.  Could you please commit the changes Johan suggested? Thanks

**Pull request closed without merging**